doc/team.md

```diff
@@ -1,71 +1,63 @@
-# Team Gallery
+# PyGMT Team
 
-We are an international team dedicated to building a Pythonic API for the
-Generic Mapping Tools (GMT). Our goal is to improve GMT's accessibility for
-new and experienced users by creating user-friendly interfaces with the GMT
-C API, supporting rich display in Jupyter notebooks, and integrating with
-the PyData ecosystem.
+We are an international team dedicated to building a Pythonic API for the Generic Mapping
+Tools (GMT). Our goal is to improve GMT's accessibility for new and experienced users by
+creating user-friendly interfaces with the GMT C API, supporting rich display in Jupyter
+notebooks, and integrating with the PyData ecosystem.
```

Comment:
I feel we can remove the sentence about our goals, because it's already mentioned at https://www.pygmt.org/dev/#project-goals:
```suggestion
We are an international team dedicated to building a Pythonic API for the Generic Mapping
Tools (GMT).
```